src/NServiceBus.Core/Transports/Msmq/MsmqTransportInfrastructure.cs

```diff
@@ -89,13 +97,12 @@ public override TransportReceiveInfrastructure ConfigureReceiveInfrastructure()
 
             var msmqSettings = settings.Get<MsmqSettings>();
 
+
```

Comment:
@bording - this is weird. I ran the r# format and it didn't catch this! Will fix!